kwave/options/simulation_options.py

```diff
@@ -102,7 +102,7 @@ class SimulationOptions(object):
     stream_to_disk: bool = False
     data_recast: Optional[bool] = False
     cartesian_interp: str = 'linear'
-    hdf_compression_level: Optional[int] = None
+    hdf_compression_options: Optional[Union[int, str]] = None
```

Comment:
The fact that this is typed as a string or an integer complicates variable handling down the line. I thought we had discussed typing this as an enum?

Reply:
I agree that enum will be nice here